src/model/Board.java

```diff
@@ -84,9 +95,13 @@ public void placeZombie(Zombies zombie, int row, int col) {
      * @param chancePerTurn: starting from 0 to chancePerTurn, if the random number matches an n value then spawn given zombie.
      */
     private void generateZombie(int chancePerTurn) {
+        // Generate random int
         int n = this.random.nextInt(chancePerTurn);
         if (n == 0 || n == 1) {
+            // Place the zombie on the board
             this.placeZombie(new Zombie(), random.nextInt(height), width - 1);
```

Comment:
overboard on the comments, the comment relating to placeZombies should be placed over the class and not when its called, same for all the other instances. also the name is pretty self explanatory

Reply:
Mostly just did this so they cant ding us for marks. But i agree